.expeditor/release_habitat.pipeline.yml

```diff
@@ -410,6 +410,17 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
 
+  - label: "Update Habitat API Docs"
```

Comment:
this should actually go in the `finish_release` pipeline since we only want this to trigger when we promote to stable.

Reply:
The supervisor raml was already in 1.0 and we decided not to update as the only difference observed was the addition of the /metrics path, which is internal.

Reply:
I will move this to the correct pipeline.